lib/db/client-applications-store.js

```diff
@@ -18,7 +20,7 @@ const mapRow = row => ({
     createdAt: row.created_at,
     updatedAt: row.updated_at,
     description: row.description,
-    strategies: row.strategies,
+    strategies: fromText(row.strategies) || '',
```

Comment:
I take it for compatibility you've opted for fromText instead of JSON support handled by the driver?

Reply:
Yeah, the `fromText` is a quick wrapper. The Postgres and Mysql drivers handle JSON objects differently -- some of them return the JSON text by default, others parse it into a JS object for you. So this was just a tool to use to make sure it gets deserialized into an object, since that's what the services and other methods expect.

Reply:
In that case is the `|| ''` not supposed to be ` || {}`?